tests/endpoints/test_schemas.py

```diff
@@ -85,33 +85,26 @@ class SchemaExample(schemas.Model):
 def test_string_list_type():
     class SchemaExample(schemas.Model):
 
-        area_list = schemas.StringListType(schemas.StringModelType(schemas.Area), separator="+")
         string_list = schemas.StringListType(schemas.StringType, separator="+")
 
-    string_data = {"string_list": "a+b+c", "area_list": "10km@-36.847585,174.765742+10km@-41.288058,174.778265"}
+    string_data = {"string_list": "a+b+c"}
```

Comment:
Any specific reason for removing the list? Applies below as well.

Reply:
yep it was due to me removing the area class will bring back the previous area test cases.